File: Data Structures & Algorithms/max-consecutive-ones/submission-0.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        num = 0
        max_num = 0
        for i in nums: 
            if(i == 1):
                num = num +1
            else: 
                num = 0
            if(num > max_num):
                    max_num = num
        return max_num
            
        

File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        HashMap<Integer, Integer> dupMap = new HashMap();
        if (nums.length == 0)
            return false;
        for (int i = 0; i < nums.length; i++) {
            if (dupMap.containsKey(nums[i])) {
                return true;
            } else {
                dupMap.put(nums[i], i);
            }
        }
        return false;
    }
}